### DISCUSSION TITLE
harvest don't fetch files

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3852] [General]
hello everyone, I'm doing a harvest but when I run it, it always stays in this state and it doesn't change and I run several drush commands but it's still the same, I don't know what it could be or if I should run something else later I'll be attentive thank you very much

![image](https://user-images.githubusercontent.com/22879539/195427197-265b5b18-acee-4ca4-acec-0caf3369efe9.png)

**Comment by janette** [comment]
@tridoxx what do you get with `drush queue-list`?
Have you run cron?

**Reply by tridoxx** [reply]
hi @janette  thanks for reply, i believe that dkan enter in a error, i tried to reinstall it and now show me this , triying to do a harvest, i am using this command `drush dkan:harvest:register --identifier=example2 --extract-uri=http://3.20.221.84/prueba_organizada1_Dataset.json`
![image](https://user-images.githubusercontent.com/22879539/195440823-0bfa48bd-6f52-4a6d-a1ce-74b85b748f6f.png)

and when i tried to generate sample-content show me this, you know any way to check about this errors?. i tried to generate sample data when is installed. 
![image](https://user-images.githubusercontent.com/22879539/195440933-8b993104-4719-448e-b4cf-2fd81075e6d3.png)

and about your questions not, i don't run a cron, and drush queue-list, show this 
![image](https://user-images.githubusercontent.com/22879539/195441358-712a806a-5721-462b-a869-982850e0ffcd.png)

**Reply by tridoxx** [reply]
update: okay idk why, but now is working restart the machine, reinstall the dkan and now work,
![image](https://user-images.githubusercontent.com/22879539/195444262-ab4998a7-f0b9-45ce-922e-846074ecdd9e.png)

**Comment by janette** [comment]
@tridoxx did you get your example1 harvest to import as expected?

**Reply by tridoxx** [reply]
yep i did, thanks for help :)
